airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -173,7 +173,7 @@ def format_datetime(
     )
     if format == "%s":
         return str(int(dt_datetime.timestamp()))
-    elif format == "%ms":
```

Comment:
I'm confused about this change ; shouldn't we continue supporting `%ms`? Doesn't that indicate that there is a bug in the formating for epoch millis as it would return an epoch microseconds string representation?

Reply:
it was added in https://github.com/airbytehq/airbyte-python-cdk/pull/373/files

but then I realised that we already have "%ms" placeholder that represents **milli**seconds instead of **micro**seconds.
To avoid confusion I decided to name them explicitly: `%epoch_microseconds`

Reply:
Would it help maintenance and reduce confusion if `format_datetime` would call `DatetimeParser.format`? It would also ensure that the same formatting capabilities are the same everywhere

Reply:
agreed, refactored